src/main/java/frc/robot/commands/teleop/conveyor/ConveyorStandby.java

```diff
@@ -1,40 +1,39 @@
-package frc.robot.commands.teleop;
+package frc.robot.commands.teleop.conveyor;
 
-import frc.robot.subsystems.Drivetrain;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.controls.Driver;
+import frc.robot.Constants;
 import frc.robot.Robot;
 
-public class TeleopDrivetrain extends CommandBase
+public class ConveyorStandby extends CommandBase
 {
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
-  private final Drivetrain drivetrain;
+  private Robot robot;
 
-  public TeleopDrivetrain(Robot robot)
+  public ConveyorStandby(Robot robot)
   {
-    drivetrain = robot.drivetrain;
-    addRequirements(drivetrain);
+    this.robot = robot;
+    addRequirements(robot.conveyor);
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize()
   {
-    drivetrain.stop();
+
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute()
   {
-    drivetrain.move(Driver.getThrottle(), Driver.getSteer());
+    robot.conveyor.standby(robot.driver.getSteerButton(Constants.CHARGE_SHOOTER));
```

Comment:
Why do we need to pass the `CHARGE_SHOOTER` button here?

Reply:
This doesn't actually tell us if the shooter is charging or not.  The shooter subsystem should be able to tell you that.  Maybe set a variable when the shooter is spinning and set it back to `false` when it stops.